examples/wearable/UIComponents/contents/user-inputs-elements/pickers/custom-picker.html

```diff
@@ -7,10 +7,12 @@
 
 	<link rel="stylesheet"
 			href="../../../lib/tau/wearable/theme/default/tau.css">
-	<link rel="stylesheet" media="all and (-tizen-geometric-shape: circle)"
+	<link rel="stylesheet"
+			media="all and (-tizen-geometric-shape: circle)"
```

Comment:
just curious. was it intended to change the line?

Reply:
It was just copy/paste effect :) . It was not intended. I am trying to prepare this PR to review now.